core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.java

```diff
@@ -1024,6 +1024,7 @@ protected void openFullScreen() {
     toolbarContainer.setVisibility(View.GONE);
     bottomToolbar.setVisibility(View.GONE);
     exitFullscreenButton.setVisibility(View.VISIBLE);
+    exitFullscreenButton.getBackground().setAlpha(60);
```

Comment:
`60` isn't 60% of 255, ie what the old xml code was doing. The alpha apis are a bit terrible